website/client/components/inventory/equipment/index.vue

```diff
@@ -234,6 +270,18 @@ export default {
     equipItem (item) {
       this.$store.dispatch('common:equip', {key: item.key, type: this.costume ? 'costume' : 'equipped'});
       this.gearToEquip = null;
+    },
+    unequipGear () {
+      console.log(this.activeItems);
+      for (let key in this.activeItems) {
+        console.log(this.activeItems[key]);
+        if (!this.activeItems[key].endsWith('base_0')) {
+          this.$store.dispatch('common:equip', {key: this.activeItems[key], type: this.costume ? 'costume' : 'equipped'});
```

Comment:
sadly this doesn't work if you have more than one gear equipped:

the website quickly fires multiple API-calls to the server, those nearly reaches the server simultaneously, so what happens next:

```
1st request comes , gets the current gear, remove.....
   2nd request comes in, gets the current gear, removes... 
      3rd request comes in, gets the current gear, removes it, saves to db
   removes it and saves to db
removes it and saves to db
```

so at the end, only one gear was unequipped the rest is still equipped, only the website thinks "yup all unequipped" 

to fix it this, probably an own api-call would be needed to unequip all gear / costume